eng/.docsettings.yml

```diff
@@ -115,6 +115,7 @@ known_content_issues:
   - ['sdk/core/Microsoft.Azure.Core.NewtonsoftJson/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
   - ['sdk/core/Microsoft.Azure.Core.Spatial.NewtonsoftJson/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
   - ['sdk/core/Microsoft.Azure.Core.Spatial/README.md', 'https://github.com/Azure/azure-sdk-tools/issues/404']
+  - ['sdk/core/System.ClientModel/README.md', 'Opt out of sections: https://github.com/Azure/azure-sdk-tools/issues/404']
```

Comment:
Should we log a follow-up issue to remove this and address the missing sections, or are we working with the understanding that this isn't being treated as a client library?

Reply:
It won't be treated as a client library so this will always be needed.

Reply:
@jsquire, I think the opt-out line links to an issue tracking this.  If we want to add an issue to track adding sections to ClientModel and Azure.Core we can do that.

Reply:
@annelo-msft:  I don't think that we need to track if our intent is that we're not going to include some sections and will always need this opt-out.   _(I agree with the statement that some of the sections just don't fit for these libraries.)_